src/routes/posts.js

```diff
@@ -172,7 +212,7 @@ const updatePost = async (req, res) => {
 // Routes
 
 router.route('/:id').delete(deletePost).patch(updatePost).get(getPost)
-
+router.route('/:postId/comment').post(createComment)
```

Comment:
sure

Reply:
> on second thought, `postId` should just be `id` for consistency with the route on top

I have made the change and pushed

Reply:
@jocrah , i woke up this morning to see merge conflicts, please review this for me so it can be merged to main as well

Reply:
okay @beblicarl, will do. but first kindly exclude `src/routes/index.js` `package-lock.json` from your changes. 

`package-lock.json` because it should mostly only change if you have a change in your `package.json`

`src/routes/index.js` because it's just a newline addition which i think we can do without

Reply:
i had to make changes in my `package-lock.json` and `package.json` because i had to install new packages in the main. The same goes for `src/routes/index.js`

Reply:
yea there's no change from the `package.json` in your pr so no need to push package-lock.json. Your package-lock.json likely changed because we're all likely using different node/npm versions . 

and i don't get the `The same goes for src/routes/index.js` since your previous statement was about installing new packages

Reply:
like always if it is not related to the pr, lets try hard not to add it to the changes. 

it will only cause conflicts for another person who is actually making changes in that particular file.

`git checkout origin/main src/routes/index.js package-lock.json` should do it

Reply:
this is the error I get when I try to reset those particular files

```
$ git checkout origin/main F:\Programming\teamwork-be\src\routes\index.js
fatal: F:Programmingteamwork-besrcroutesindex.js: 'F:Programmingteamwork-besrcroutesindex.js' is outside repository at 'F:/Programming/teamwork-be'
```